### ISSUE TITLE
Weird Attack bug

### ISSUE THREAD

**Issue by Callepo** [suriyun-production/mmorpg-kit-docs#582]
https://imgur.com/mzzwXvu

Players cant attack anything, and nothing can attack players..
no error message.. nothing..
u have to relog so it fixes
the way i did this is .
Lets orcs kill u, click respawn and ur bugged

@insthync

**Comment by insthync** [comment]
As I test, it's bug which occuring by safe area, I will try to fix it.

**Comment by insthync** [comment]
Fixed, problem occuring on host with ShooterPlayerCharacterController only, because I try to optimize raycast to find target by disable character's colliders and re-enable after finishes raycasts. So I have to remove it.

**Comment by Callepo** [comment]
@insthync amazing, Thank you

Question .
is this the correct way to start up server on linux build?
/root/TestServer/TestServer.x86_64 -startDatabaseServer -startCentralServer -startChatServer -startMapSpawnServer

**Comment by insthync** [comment]
Yes

**Comment by Callepo** [comment]
tried it multiple times yesterday and didnt work..
ill rebuild kit with the mapserver fix and see what happend.
i couldnt login or create account nothing

**Comment by insthync** [comment]
Did you open ports yet ?

**Comment by Callepo** [comment]
i have run multiple server on my vps even my game.. which was version 1.57b and all worked perfectly fine.. just new kit on 1.58 didnt start up..
but im testing it now with a new build after the mapspawnserver code fix and see what happens

**Comment by Callepo** [comment]
![image](https://user-images.githubusercontent.com/7702658/92416611-a5da7a00-f15e-11ea-8e47-1d4ad5f2ffba.png)
 
When i start server i see this

**Comment by Callepo** [comment]
This is what happens when i try to connect
https://imgur.com/9ZEJIfD

**Comment by insthync** [comment]
Please send me log files.

**Comment by Callepo** [comment]
done

**Comment by insthync** [comment]
It can't find `Build.exe` did you use config file to set `spawnExePath` ?

**Comment by Callepo** [comment]
that one was without config files, cuase i was trying the mapspawn fix.. but i tried with config now also and it does the same sec ill send u a video of the setup in mail

**Comment by insthync** [comment]
If you're going to not use config file, you must set `spawnExePath` when run the server to make it can find exe file correctly
So run command will be
```
/root/TestServer/TestServer.x86_64 -startDatabaseServer -startCentralServer -startChatServer -startMapSpawnServer -spawnExePath "/root/TestServer/TestServer.x86_64"
```

**Comment by Callepo** [comment]
yea sorry i forgot bout that 
/root/TestServer/TestServer.x86_64 -startMapSpawnServer -spawnExePath  "/root/TestServer/TestServer.x86_64" -startCentralServer -startChatServer -startDatabaseServer -machineAddress "185.95.13.129"

this what i use for no config files

**Comment by insthync** [comment]
Yep it should connectable

**Comment by Callepo** [comment]
https://youtu.be/1mu4i75lnho

I uploaded a video of the config files +  trying multiple Command lines 
If u wanna check deeper i can give u acces to my vps or u can join my Teamviewer np

**Comment by Callepo** [comment]
[Log_Central_MapSpawn_Chat_Database.txt](https://github.com/suriyun-production/mmorpg-kit-docs/files/5184975/Log_Central_MapSpawn_Chat_Database.txt)
[Log_Map(Map001) Instance().txt](https://github.com/suriyun-production/mmorpg-kit-docs/files/5184976/Log_Map.Map001.Instance.txt)
[Log_Map(Map002) Instance().txt](https://github.com/suriyun-production/mmorpg-kit-docs/files/5184977/Log_Map.Map002.Instance.txt)

New log files after trying clean upload again 1 run without config files and right commandline and 1 time with config files and right command line
it even produced the Map logs but still cant connect

**Comment by insthync** [comment]
About attack bug, it was fixed in 1.58b, I also try to fix issues with MMO servers, try it.
